<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClient</class>
 <widget class="QDialog" name="AddClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>161</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_phone">
      <property name="text">
       <string>Telefone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phone"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>341</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_email">
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="email"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Informações pessoais</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Endereço</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>161</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_cep">
      <property name="text">
       <string>CEP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="address_cep"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>341</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_street">
      <property name="text">
       <string>Rua</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="address_street"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>241</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_city">
      <property name="text">
       <string>Cidade</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="address_city"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>121</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_uf">
      <property name="text">
       <string>UF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="address_uf"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>131</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="label_address_number">
      <property name="text">
       <string>Número</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="address_number"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>131</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QLabel" name="label_complement">
      <property name="text">
       <string>Ap/Bloco*</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="address_optional"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>430</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>* Opcional</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>341</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_name">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Nome Completo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>161</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_cpf">
      <property name="text">
       <string>CPF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cpf"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>AddClient</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
